src/cpptools.ts

```diff
@@ -208,17 +208,28 @@ export class CppConfigurationProvider implements cpt.CustomConfigurationProvider
         newBrowsePath.push(includePathItem);
       }
     }
+
+    let cpptoolsCompilerPath = (flags.length === 0) ? comp_path : `${comp_path} ${flags.join(' ')}`;
+    if (sysroot && cpptoolsCompilerPath.indexOf("--sysroot") < 0) {
+      if (sysroot.match(/\s/)) {
+        cpptoolsCompilerPath += ` "--sysroot=${sysroot}"`;
```

Comment:
The opening `"` seems to be in the wrong place.  Shouldn't it be after `--sysroot=`?  And is `sysroot` guaranteed not to have additional `"`'s in it?

Reply:
Thank's for looking over this. Think this should be ok. The `sysroot` option points to an path. The path will be quoted if it contains a space. The double quote surrounds `--sysroot` so the whole term, option and path, is interpreted as an command line argument.